kinto/core/authorization.py

```diff
@@ -221,6 +221,9 @@ def _find_required_permission(self, request, service):
         method = request.method.lower()
         required_permission = self.method_permissions.get(method)
 
+        # For create permission, the object id is the plural endpoint
```

Comment:
Do you like ending comments with a `.`?

I'm not sure why, but this comment sounds a bit confusing to me. Maybe something like
`The create permission object id is the one for the plural endpoint.`?